apps/api/src/domain/models/playlist/entities/playlist.from-event-history.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { LanguageCode } from '@coscrad/api-interfaces';
+import { NotFound } from '../../../../lib/types/not-found';
+import { TestEventStream } from '../../../../test-data/events';
+import { AggregateType } from '../../../types/AggregateType';
+import buildDummyUuid from '../../__tests__/utilities/buildDummyUuid';
+import { PlaylistCreated } from '../commands/playlist-created.event';
+import { PlaylistNameTranslated } from '../commands/translate-playlist-name/playlist-name-translated.event';
+import { Playlist } from './playlist.entity';
+
+const playlistId = buildDummyUuid(12);
+
+const playlistNametext = 'the playlist name';
+
+const originalLanguageCode = LanguageCode.English;
+
+const translationLanguageCode = LanguageCode.Chilcotin;
+
+const aggregateCompositeIdentifier = {
+    type: AggregateType.playlist,
+    id: playlistId,
+};
+
+const playlistCreated = new TestEventStream().andThen<PlaylistCreated>({
+    type: `PLAYLIST_CREATED`,
+    payload: { name: playlistNametext, languageCodeForName: originalLanguageCode },
+});
+
+const playlistNameTranslated = playlistCreated.andThen<PlaylistNameTranslated>({
+    type: `PLAYLIST_NAME_TRANSLATED`,
+    payload: { text: playlistNametext, languageCode: translationLanguageCode },
```

Comment:
Don't make the translation of the name the same as the name itself. 
```ts
const playlistNameTranslation = "Translated Name";
// ...
payload: { text: playlistNameTranslation, languageCode: ...}
```

Reply:
done